### ISSUE TITLE
Incomplete code files?

### ISSUE THREAD

**Issue by vaibhavvarshney0** [voidism/L2KD#1]
It seems some code files are missing from this repo like train.sh, test.sh etc. or am I missing something?

**Comment by voidism** [comment]
Hi, sorry that I overlooked this. I just uploaded the missing files in the latest commit  14a5df3.

**Comment by vaibhavvarshney0** [comment]
Thanks.
Also 
For **distil to conduct Word-KD** do we have to train respective teacher model separately and provide it?

**Comment by voidism** [comment]
Yes, we need to train a single-task teacher model separately before we perform Word-KD.
